### PR TITLE
[IBM] Update openshift kubeflow installation

### PR DESCRIPTION
**Pull request by moficodes**
* Target commit `552a4ba84567ed8c0f9abca12f15b8eed000426c` for working tekton installtion
* Update argo installation url.

### CODE REVIEW

**Comment by moficodes** [comment]
@Tomcli

File: content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md [hunk 1]
```diff
@@ -52,15 +52,22 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
...
 export CONFIG_FILE=kfctl_ibm.yaml
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.master.kfptekton.yaml"
+export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/distributions/kfdef/kfctl_openshift.master.kfptekton.yaml"
 
 # Generate Kubeflow:
 mkdir -p ${KF_DIR}
 cd ${KF_DIR}
-curl -L ${CONFIG_URI} > ${CONFIG_FILE}
+
+wget ${CONFIG_URI} -O kfctl.yaml
```
Comment (Tomcli) [inline content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md:61]: should be change kfctl.yaml to `${CONFIG_FILE}`?

Comment (moficodes) [reply content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md:61]: making that change and pushing again.

File: content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md [hunk 2]
```diff
@@ -52,15 +52,22 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
...
 export CONFIG_FILE=kfctl_ibm.yaml
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.master.kfptekton.yaml"
+export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/distributions/kfdef/kfctl_openshift.master.kfptekton.yaml"
 
 # Generate Kubeflow:
 mkdir -p ${KF_DIR}
 cd ${KF_DIR}
-curl -L ${CONFIG_URI} > ${CONFIG_FILE}
+
+wget ${CONFIG_URI} -O ${CONFIG_URI}
```
Comment (Tomcli) [inline content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md:61]: oh it should be config_file
```suggestion
wget ${CONFIG_URI} -O ${CONFIG_FILE}
```

File: content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md [hunk 3]
```diff
@@ -52,15 +52,22 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
...
+
+# On MacOS
+sed -i '' -e 's#https://github.com/kubeflow/manifests/archive/master.tar.gz#https://github.com/kubeflow/manifests/archive/552a4ba84567ed8c0f9abca12f15b8eed000426c.tar.gz#g' ${CONFIG_URI}
+
+# On Linux
+sed -i -e 's#https://github.com/kubeflow/manifests/archive/master.tar.gz#https://github.com/kubeflow/manifests/archive/552a4ba84567ed8c0f9abca12f15b8eed000426c.tar.gz#g' ${CONFIG_URI}
 
 # Deploy Kubeflow. You can customize the CONFIG_FILE if needed.
-kfctl apply -V -f ${CONFIG_FILE}
+kfctl apply -V -f ${CONFIG_URI}
```
Comment (Tomcli) [inline content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md:70]:
```suggestion
kfctl apply -V -f ${CONFIG_FILE}
```

**Pull request merged**